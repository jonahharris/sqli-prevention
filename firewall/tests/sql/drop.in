>>> Sada testovacich SQL dotazu: drop
DROP DATABASE authors;
DROP SCHEMA authors;
DROP DATABASE IF EXISTS authors;
DROP SCHEMA IF EXISTS authors;
