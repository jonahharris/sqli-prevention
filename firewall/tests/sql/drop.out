>>> Sada testovacich SQL dotazu: drop
DATABASE authors;DROP 0;STMT;1;0;
DATABASE authors;DROP 0;STMT;1;0;
DATABASE authors;DROP 1;STMT;1;0;
DATABASE authors;DROP 1;STMT;1;0;
